prow.sh

```diff
@@ -1066,15 +1066,21 @@ main () {
                 docker tag "$i:latest" "$i:csiprow" || die "tagging the locally built container image for $i failed"
             done
 
-            if [ -e deploy/kubernetes/rbac.yaml ]; then
+            if [ -e "deploy/kubernetes/rbac.yaml" ] || [ -e "deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml" ]; then
```

Comment:
I think we will have other repos in the future that contain multiple binaries with different rbac files. Do you see a way that we can make the path configurable so that it will work for other repos?

Maybe we loop over $cmds and try to find rbac files that match?

Reply:
@msau42 Updated this to find an RBAC file in the format `rbac-${cmd}.yaml`

Is this a format we can assume? I could generalize it to allow for `${cmd}-rbac.yaml` and other cases, but I'm not sure how far we want to go in generalizing it.

Reply:
I think this format is fine. Thanks!